tools/wopiopen.py

```diff
@@ -11,21 +11,24 @@
 # usage function
 def usage(exitcode):
   '''Prints usage'''
-  print('Usage : ' + sys.argv[0] + ' [-h|--help] <filename> <uid> <gid>')
+  print('Usage : ' + sys.argv[0] + ' [-h|--help] [-m|--mode 1|2|3] <filename> <uid> <gid>')
   sys.exit(exitcode)
 
 # first parse the options
 try:
-  options, args = getopt.getopt(sys.argv[1:], 'hv', ['help', 'verbose'])
+  options, args = getopt.getopt(sys.argv[1:], 'hvm:', ['help', 'verbose', 'mode'])
 except getopt.GetoptError as e:
   print(e)
   usage(1)
 verbose = False
+mode = 3   # 3 is full access, 2 for read-only, 1 for view-only.
 for f, v in options:
```

Comment:
What does f stand for?

Reply:
flag and value. This is boiler-plate code that I copy over every time I have to parse some command line arguments...

Reply:
These single letters 🤯 I'd prefer if we wrote flag and value to make to code easier to understand by a quick look. But I'm also all for democracy so if I'm the minority, let's continue with this.